### ISSUE TITLE
Kyma admin insufficient permissions

### ISSUE THREAD

**Issue by Demonsthere** [kyma-project/kyma#3780]
**Description**

The default kyma-admin user (and role) has insufficient permissions to view/list resources on a kyma cluster

**Expected result**

The kyma-admin has access to all kyma managed resources

**Actual result**

Resources from the api `*.knative.dev` are not included in the admin clusterrole

**Steps to reproduce**

`kubectl get all --all-namespaces`

**Troubleshooting**

```
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=podautoscalers, subresource=) (get podautoscalers.autoscaling.internal.knative.dev)
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=images, subresource=) (get images.caching.internal.knative.dev)
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=clusterchannelprovisioners, subresource=) (get clusterchannelprovisioners.eventing.knative.dev)
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=channels, subresource=) (get channels.eventing.knative.dev)
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=subscriptions, subresource=) (get subscriptions.eventing.knative.dev)
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=clusteringresses, subresource=) (get clusteringresses.networking.internal.knative.dev)
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=configurations, subresource=) (get configurations.serving.knative.dev)
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=revisions, subresource=) (get revisions.serving.knative.dev)
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=services, subresource=) (get services.serving.knative.dev)
Error from server (Forbidden): Forbidden (user=admin@kyma.cx, verb=list, resource=routes, subresource=) (get routes.serving.knative.dev)
```

**Comment by piotrmsc** [comment]
@abbi-gaurav you might be interested in this one, should users be able to list those crds?

**Comment by abbi-gaurav** [comment]
@piotrmsc for the eventing related, I would say yes for only read permissions.
For serving and autoscaling, I would say @joek can provide better feedback.

**Comment by sjanota** [comment]
@abbi-gaurav @joek More important question is if you want the fix to end up in 1.0.

**Comment by piotrmsc** [comment]
@abbi-gaurav @joek if you think it should be available please adjust built-in  RBAC roles in the kyma with corresponding api groups and verbs
